swap_meet/vendor.py

```diff
@@ -1,2 +1,73 @@
 class Vendor:
-    pass
+    def __init__(self, inventory=None, item=None):
+        self.inventory = inventory if inventory is not None else []
+        self.item = item
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
+        else:
+            for i in range(len(self.inventory)):
+                inventory_set = (self.inventory[i], i)
+                if item in inventory_set:
+                    self.inventory.pop(i)
+                    break
+            return inventory_set[0]
+    
+    def get_by_category(self, category=''):
+        categories_present_list = []
+        for i in range(len(self.inventory)):
+            if category == self.inventory[i].category: 
+                categories_present_list.append(self.inventory[i])
+            else:
+                continue
+        return categories_present_list
+
+    def swap_items(self, friend, my_item, their_item):
+        if my_item in self.inventory and their_item in friend.inventory:
+            remove_my_item = self.remove(my_item) 
+            friend.add(remove_my_item) 
+
+            remove_friend_item = friend.remove(their_item)
+            self.add(remove_friend_item)
```

Comment:
Here you don't necessarily have to create variables you can just do 
 ```suggestion
            self.remove(my_item) 
            friend.add(my_item) 

            remove_friend_item = friend.remove(their_item)
            self.add(their_item)
```